dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1200,6 +1200,15 @@ def browser_request(url:, method: 'GET', headers: {}, body: nil, code: 200, trie
   GHERKIN
 end
 
+And(/^I submit this gamelab level$/) do
+  steps <<~GHERKIN
+    And I press "runButton"
+    And I wait to see "#submitButton"
+    And I press "submitButton"
+    And I wait for 10 seconds
```

Comment:
I used "wait for x seconds" here to allow enough time for the project to be submitted.